ptx/lib/zluda_ptx_impl.cpp

```diff
@@ -1406,17 +1436,21 @@ extern "C"
                     if (specifier_with_length)
                     {
                         s += specifier_with_length;
-                    }
-                    if (len > 0)
-                    {
-                        uint64_t value = read_valist(valist_ptr, valist_offset, len);
-                        handle = __ockl_printf_append_args(handle, 1, value, 0, 0, 0, 0, 0, 0, 0);
+                    } else {
+                        s++;
+                        continue;

```

Comment:
This is the case of unkown/invalid specifier, e.g. `%!`. Shouldn't this be a `break` instead?

Reply:
Can't do that, as it'll make the current oversimplified handling of sub-specifier (aka. ignoring them and assuming they are valid) break:
```
Characters: a A 
Decimals: 1977 650000
Preceding with blanks: Preceding with zeros: Some different radices: 100 64 144 floats: 3.14 Width trick:    10 
A string
```

While using continue; here breaks unknown/invalid specifier handling, considering we've implemented the entirety of the standard specifiers + handling of invalid specifiers aren't strictly defined, I find it a good compromise

Reply:
Ah, good point